.gitignore

```diff
@@ -6,3 +6,5 @@ build
 dist
 package-lock.json
 yarn.lock
+test/fixtures/basic-re/lib
```

Comment:
why this one is ignored? does it contain some compiled files?

Reply:
It’s the downloaded lib when running the compilation for the first time